sled-agent/src/instance.rs

```diff
@@ -557,6 +573,8 @@ impl Instance {
     async fn stop(&self) -> Result<(), Error> {
         let mut inner = self.inner.lock().await;
 
+        // XXX how to free the static V6 address? where do I store it?
+
```

Comment:
Couple options:

- Could just be a call to the Nexus endpoint
- Alternatively, the allocation for these addresses could be done by Nexus, rather than the sled agent?

Reply:
this was removed - Nexus should free it during instance delete.